core/remoteviewserver.cpp

```diff
@@ -228,5 +228,6 @@ void RemoteViewServer::clientConnectedChanged(bool connected)
 void RemoteViewServer::requestUpdateTimeout()
 {
     emit requestUpdate();
+    emit frameRequested();
```

Comment:
Yep, though the signal is not part of the interface it's why I added a new one.
In the end I can indeed use this one using Qt 4 style connection.
Fixed